render-wasm/src/images.rs

```diff
@@ -1,3 +1,51 @@
+use crate::math;
 use skia_safe as skia;
 
 pub type Image = skia::Image;
+
+pub fn draw_image_contain(
```

Comment:
I don't understand this function name (maybe `draw_image_in_container`? if the container is a clip